lib/atomic_web/components/socials.ex

```diff
@@ -0,0 +1,29 @@
+defmodule AtomicWeb.Components.Socials do
+  @moduledoc false
+  use Phoenix.Component
+
+  def socials(assigns) do
+    ~H"""
+    <%= if @entity.socials do %>
+      <div class="mt-4 flex gap-4">
+        <%= for {social, icon, url_base} <- [
+            {:tiktok, "tiktok.svg", "https://tiktok.com/"},
+            {:instagram, "instagram.svg", "https://instagram.com/"},
+            {:facebook, "facebook.svg", "https://facebook.com/"},
+            {:x, "x.svg", "https://x.com/"}
+          ] do %>
```

Comment:
Socials list should be configured inside the socials component, please define an outside function or variable which returns the socials.

```suggestion
      <div class="mt-4 flex gap-4">
        <%= for {social, icon, url_base} <- [
            {:tiktok, "tiktok.svg", "https://tiktok.com/"},
            {:instagram, "instagram.svg", "https://instagram.com/"},
            {:facebook, "facebook.svg", "https://facebook.com/"},
            {:x, "x.svg", "https://x.com/"}
          ] do %>
```